keymap.dtsi

```diff
@@ -25,10 +25,16 @@
   #define _C         LG
   #define _REDO      _C(LS(Z))
   #define _POWER     K_POWER
+  #define _W         LA
+  #define _HOME      LG(LEFT)
```

Comment:
Let's define these in terms of `_C` as well instead of `LG`, to be consistent with the non-macOS definitions.  For example, I'd like:

```c
#define _HOME _C(LEFT)
```

Reply:
Makes sense. Edited!